axis.py

```diff
@@ -63,15 +64,14 @@ class Axis360API(object):
 
     log = logging.getLogger("Axis 360 API")
 
-    def __init__(self, _db, collection):
+    def __init__(self, collection):
```

Comment:
I changed all the API constructors to just take a Collection and acquire the database connection from it using Session.object_session().